src/tracing/json/json_tracing.cpp

```diff
@@ -63,12 +63,25 @@ ::Json::Value GetPayload(PayloadDecoderType & decoder)
     PayloadEntry entry;
     StringBuilder<128> formatter;
 
+    int unique_idx = 0;
+
     while (decoder.Next(entry))
     {
         switch (entry.GetType())
         {
         case PayloadEntry::IMPayloadType::kNestingEnter:
             formatter.Reset().Add(entry.GetName()); // name gets destroyed by decoding
+
+            while (value.isMember(formatter.c_str()))
```

Comment:
This code is not very clear....  Please document that this is trying to find a name that is not already being used as a key in "value"?